RecoParticleFlow/PFProducer/plugins/importers/GeneralTracksImporter.cc

```diff
@@ -15,6 +15,9 @@ class GeneralTracksImporter : public BlockElementImporterBase {
       : BlockElementImporterBase(conf, sumes),
         src_(sumes.consumes<reco::PFRecTrackCollection>(conf.getParameter<edm::InputTag>("source"))),
         muons_(sumes.consumes<reco::MuonCollection>(conf.getParameter<edm::InputTag>("muonSrc"))),
+        trackQuality_((conf.existsAs<std::string>("trackQuality"))
+                          ? reco::TrackBase::qualityByName(conf.getParameter<std::string>("trackQuality"))
+                          : reco::TrackBase::qualityByName("highPurity")),
```

Comment:
Here and elsewhere, this could be just
```suggestion
                          : reco::TrackBase::highPurity),
```